pagopa/pn-frontend
# fix(pn-3812): adds disable behaviour for edit button special contacts

## Short description
Adds disable behaviour for other edit buttons in special contacts while editing one.

## List of changes proposed in this pull request
- Adds disable behaviour for other edit buttons in special contacts while editing one.

## Screenshot
<img width="1110" alt="Screenshot 2023-02-16 at 12 24 44" src="https://user-images.githubusercontent.com/100523847/219352638-def6d15c-f0d9-44e8-bfd9-2a5b59615a7d.png">

## How to test
Try to edit a field in the 'Altri recapiti' section in page 'I Tuoi Recapiti', the other 'Modifica' buttons should be disabled
